vispy/plot/plotwidget.py

```diff
@@ -243,13 +243,19 @@ def mesh(self, vertices=None, faces=None, vertex_colors=None,
         meshdata : MeshData | None
             Meshdata to use. If not None, then vertices, faces, and fname
             must be None.
+        shading : str | 'auto'
```

Comment:
'auto' is a `str`, so no need to put it up here

docstring should be more like (please fill in missing values if you can):

Shading to use, can be XXX, YYY, or ZZZ. Default ('auto') will use None if face_colors is set, and 'smooth' otherwise.